handbook/product/product_documents.md

```diff
@@ -0,0 +1,65 @@
+
+These documents were created out of a need to orient around a particular problem statement, and the realization that it sometimes takes multiple [RFCs](../communication/rfcs.md) to fully solve a problem. This helps to aggregate the motivations and learnings around a problem, so that individual RFCs can focus on a particular proposed solution.
+
+_All PDs are in a [public Google Drive folder](https://drive.google.com/drive/folders/1Wd-Xx2wNbFtSzeJwbZqMOxdbFDUFxlyR)._
+
+## PDs have similar properties to RFCs
+
+They are:
+
+- [Sequentially numbered](../communication/rfcs.md#rfcs-are-sequentially-numbered)
+- [Google Docs](../communication/rfcs.md#rfcs-are-google-docs)
+- [Public](../communication/rfcs.md#rfcs-are-public)
+- [Open to external contributions](../communication/rfcs.md#external-contributors)
+
+## Status
+
+Each PD has a status that is in the title of the PD (e.g. "PD 1 WIP: Title"). The author is responsible for keeping the status updated.
+
+| Status | Description |
+|-------|-------------|
+| WIP | The author is still drafting the PD and it is not ready for review. |
+| REVIEW | The PD is ready to be reviewed. The PD explicitly lists whose approvals are required and a requested timeline for those approvals. |
+| DEFINED | The problem statement defined and agreed upon, and is locked to changes. This is to ensure that anyone involved in solving the problem statement defined within the PD does not need to worry about changing scope. A change to the problem statement likely requires a new PD to be created to define the new problem statement. If the update is minor enough, all interested parties should be notified. |
+| ABANDONED | There are no plans to move forward with solving the problem statement defined in the PD. The particular reason is communicated in the metadata section of the PD. For example, the PD may have failed to get the necessary approvals, it may be been superseded by another PD, priorities may have changed, or we may not have resources to work on this PD in the foreseeable future. |
+| SOLVED | The problem statement defined in the PD has been solved. |
+
+## PD structure
+
+Effective PDs contain the following information. The below might feel like a heavy structure, this is intentional to help ensure all important considerations and context has been written down. Many sub-sections are very short, and some can be omitted if not relevant to the specific PD.
+
+_For convenience, there is a [Google Docs Template](https://docs.google.com/document/d/1MBZxnRlDG69Fyvzpai5rBqxizvX5zVeZiUe6z7VZrjk/edit?usp=sharing) that can be used when creating new PDs. The template format should help clarify the below structure._
```

Comment:
I am wondering if the template should be the source of truth for what sections are required and what each means (instead of duplicating it here in the handbook and in the template). I think in practice people will use the template and the handbook won't be a good source of truth for what belongs in a PD.

Reply:
I was wondering about that too. Perhaps a more high level discussion of why each of the top sections is important and why we included them in the PD template would be more helpful here. Then the template becomes the source of truth for all the individual sections and descriptions/prompts.